src/array.jl

```diff
@@ -735,3 +735,40 @@ summary(A::CategoricalArray{T, N, R}) where {T, N, R} =
 
 refs(A::CategoricalArray) = A.refs
 pool(A::CategoricalArray) = A.pool
+
+"""
+    sort!(V::CategoricalVector; [rev::Bool = false])
```

Comment:
No need for brackets, `=` already implies it's optional. Also, the Base docstring doesn't use spaces around `=`, and `v` is in lower case. Same for `sort`.